packages/wonder-blocks-modal/components/modal-content.js

```diff
@@ -26,7 +26,7 @@ export default class ModalContent extends React.Component<Props> {
 
     static __IS_MODAL_CONTENT__ = true;
     static isClassOf(instance: any) {
-        return instance && instance.type && instance.type.__IS_MODAL_CONTENT__;
+        return instance?.type?.__IS_MODAL_CONTENT__;
```

Comment:
What would be the reason to need the first `?` here?

Reply:
Because `instance` should be typed as `React.Element<any>` like the other `isClassOf` functions.  Nice catch.  I'll make that change.

Reply:
I can't use `React.Element<any>` here because another part of the code tries to pass a `React.Node` here which can be `null`.

Reply:
I'll add a comment.